dietpi/dietpi-software

```diff
@@ -11103,6 +11109,34 @@ _EOF_
 			modprobe binfmt_misc 2> /dev/null && G_EXEC systemctl restart systemd-binfmt
 		fi
 
+		if To_Install 206 # Moonlight
+		then
+			## Install script adapted from
+			## - https://dl.cloudsmith.io/public/moonlight-game-streaming/moonlight-embedded/setup.deb.sh
+
+			# APT deps
+			aDEPS=('apt-transport-https' 'ca-certificates' 'debian-keyring' 'debian-archive-keyring' 'gnupg')
+
```

Comment:
None of these is neeeded:
```suggestion
```
- APT has native HTTPS support, so `apt-transport-https` is just a transitional dummy package since Buster: https://packages.debian.org/apt-transport-https
- `ca-certificates` is necessarily pre-installed, otherwise the script would not even run until here 😉.
- `debian-keyring` is for Debian package maintainers to upload packages, never for users/admins to install packages: https://packages.debian.org/bookworm/debian-keyring
- `debian-archive-keyring` is a hard dependency of APT on Debian, while on Raspbian it is `raspbian-archive-keyring`. So the correct one is always pre-installed: https://packages.debian.org/bullseye/apt
- `gnupg` is as well pre-installed on DietPi, otherwise a bunch of other tasks and installs wouldn't work either.

Reply:
Ah nice, I didn't know that about the keyrings. I thought it was how developers signed their packages.

> apt-transport-https, ca-certificates, gnupg

Yeah I figured, wasn't sure whether to be explicit or not